### ISSUE TITLE
Blocked commands

### ISSUE THREAD

**Issue by Someone-NEW** [NoCheatPlus/Issues#60]
Some commands are now not blocked in newest versions like /worldguard /wg and some other but not all commands are unblocked i use newst #904 jenkins build ans Spigot 1.8.8 and newest worldguard
need to be fixed blocked command all in my config is already active to block. thaks

**Comment by asofold** [comment]
Could you post/paste the protection... section of your configuration?

**Comment by Someone-NEW** [comment]
My config http://justpaste.it/px9m
and pictur of NCP version https://proxy.spigotmc.org
![215fd4cda0ca4d227d6ae949df4eef9dd712393a](https://cloud.githubusercontent.com/assets/16355900/12002610/c23c862c-ab02-11e5-8058-a1c26ea933e7.jpg)

**Comment by asofold** [comment]
So this is about the 'nopermission' section or is 'consoleonly' also an issue.

On quick testing, consoleonly seems to work as intended.

Did you update WorldGuard as well, about the same time with NCP?

**Comment by Someone-NEW** [comment]
I use worldguard-legacy-6.1.2-SNAPSHOT-dist.jar updated but befor i have not problems

**Comment by asofold** [comment]
So with consoleonly there is no problem?

**Comment by Someone-NEW** [comment]
I cant block this command i have on hide and in secion consoleonly but still unblocked

**Comment by asofold** [comment]
The affected commands have to be in both lists, no problem with commands that are only in one of the lists?

**Comment by Someone-NEW** [comment]
Yeah, especially for you, I tested all the possibilities and does not help because it is not possible to block.

**Comment by asofold** [comment]
I just asked, in order to understand :) - will do some testing then.

**Comment by Someone-NEW** [comment]
Thank you i hope find some solution
